src/Card/Card.tsx

```diff
@@ -25,6 +27,7 @@ export interface CardProps extends HTMLAttributes<HTMLElement> {
 
 export const Card: FC<CardProps> = ({
   tag = 'div',
+  href = undefined,
```

Comment:
i would use "null"

Reply:
"undefined" may also work, personal choice.

Reply:
Yes, it should work the same. I chose `undefined` because it's the most similar thing to not set the value at all.